src/string.F90

```diff
@@ -234,39 +231,38 @@ end function to_upper
 ! integers.
 !===============================================================================
 
-function zero_padded(num, n_digits) result(str)
-  integer, intent(in) :: num
-  integer, intent(in) :: n_digits
-  character(11)       :: str
-
-  character(8)        :: zp_form
-
-  ! Make sure n_digits is reasonable. 10 digits is the maximum needed for the
-  ! largest integer(4).
-  if (n_digits > 10) then
-    call fatal_error('zero_padded called with an unreasonably large &
-         &n_digits (>10)')
-  end if
-
-  ! Write a format string of the form '(In.m)' where n is the max width and
-  ! m is the min width.  If a sign is present, then n must be one greater
-  ! than m.
-  if (num < 0) then
-    write(zp_form, '("(I", I0, ".", I0, ")")') n_digits+1, n_digits
-  else
-    write(zp_form, '("(I", I0, ".", I0, ")")') n_digits, n_digits
-  end if
-
-  ! Format the number.
-  write(str, zp_form) num
-end function zero_padded
+  function zero_padded(num, n_digits) result(str)
```

Comment:
Not sure what you mean. Before it was under-spaced relative to other procedures in the module, so I just adjusted the indentation so that everything is consistent. @smharper's Fortran style checker script is not quite smart enough to catch something like this.